docs/domains/index.html

```diff
@@ -1120,11 +1144,12 @@
       "description": "DomainSpec is a description of a domain.",
       "type": "object",
       "properties": {
-        "keepJRFSchema": {
-          "description": "Keep JRF schema between lifecycle updates",
-          "type": "boolean"
+        "configuration": {
+          "description": "Models and overrides affecting the WebLogic domain configuration",
+          "$ref": "#/definitions/Configuration"
         },
         "configOverrideSecrets": {
+          "deprecated": "true",
           "description": "A list of names of the secrets for optional WebLogic configuration overrides.",
```

Comment:
For each deprecated element in the schema definition, can the description be updated to include the name of the replacement field?

Reply:
Ah - I see you already did this in other places, but missed this one...